.appveyor.yml

```diff
@@ -13,6 +13,11 @@ environment:
   NP_BUILD_DEP: "numpy==1.14.5"
   NP_TEST_DEP: "numpy==1.14.5"
   DAILY_COMMIT: master
+  # Unless there is a problem in releasing wheels, this should be set to an
+  # empty string. If there is a problem releasing wheels and a new build
+  # must be uploaded, then "--build-number <some number>" can be set
+  # https://github.com/MacPython/wiki/wiki/Build-Tags
+  BDIST_WHEEL_ARGS: "--build-number 1"
```

Comment:
Nice paragraph! Could you forward-port to master when this is done, with the empty string? =)

Reply:
I couldn't figure out how to do it for linux/mac too... I would rather not forward port until somebody finds out how ???

Reply:
I'm confused... Isn't the process exactly the same but on Travis?

Reply:
Not really, for Linux and Mac the build process is pretty obscure and jumps through many more scripts in muktibuild. Finally for Linux is gets spawned as port of an isolated environment in Dockers so you need to be careful in how you pass parameters to it.